db/migrate/20170508150135_add_foreign_keys.rb

```diff
@@ -0,0 +1,31 @@
+class AddForeignKeys < ActiveRecord::Migration
+  def change
+    add_index :notifications, [:subject_id, :subject_type]
+    add_index :order_details, :project_id
+    add_index :order_statuses, :parent_id
+    add_index :orders, :merge_with_order_id
```

Comment:
This index already exists in NU. There were a bunch of indexes that we didn't have in oracle because of it's weirdness around foreign keys, so we added them. Seems like we might have added too many.

https://github.com/tablexi/nucore-nu/commit/e3a67c1a4ee478473bee3a5ccd9182c5ffbe8e35

There's also `:price_group_members, :account_id` already.